lib/active_admin/views/components/table_for.rb

```diff
@@ -91,10 +93,10 @@ def build_table_header(col)
       end
 
       def build_table_body
-        @tbody = tbody do
+        @tbody = tbody data: @tbody_data do
           # Build enough rows for our collection
           @collection.each do |elem|
-            tr(id: dom_id_for(elem), class: @row_class&.call(elem))
+            tr(id: dom_id_for(elem), class: @row_class&.call(elem), data: @row_data&.call(elem))
```

Comment:
Good point, I switched to the `tbody_html` options, as I found more used of `*_html` than `*_html_options` in the codebase. 

For the `row_data` it's a bit more complex as we need the proc on the row element. We could add a new `row_html` method with an optional proc arguments, but we would also need to still support the `row_class` for backward compatibility. Which I guess would overwrite a `class` set by `row_html` if both options are used? What would you prefer?

Reply:
If you prefer to keep row_class support, I think it would be fine if it's present to reverse merge that into the new row_html value so the class in row_html would take precedence over row_class. Thank you.

Reply:
Added `row_html`, but kept `row_class` for backward compatibility. I think we should be good now.